src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,3 +85,118 @@ def load_arguments(self, _):
 
     with self.argument_context('cosmosdb database') as c:
         c.argument('throughput', type=int, help='Offer Throughput (RU/s)')
+
+# SQL database
+    with self.argument_context('cosmosdb sql database') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--name', '-n'], help="Database name")
+        c.argument('throughput', help='The throughput of SQL database (RU/s). Default value is 400')
```

Comment:
You can add `default=400` and remove `Default value is 400` from help, it will append the same message automatically and set throughput to 400 if the option is not provided.